onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-partners/components/knowledge-product-selector/knowledge-product-selector.component.ts

```diff
@@ -17,12 +17,29 @@ export class KnowledgeProductSelectorComponent {
 
   alertStatusMessage: string = `Partner organizations you collaborated with or are currently collaborating with to generate this result. <li>Please note that CGIAR Centers are not listed here. They are directly linked to <a class="open_route" href="/result/result-detail/${this.resultCode}/theory-of-change?phase=${this.versionId}" target="_blank">Section 2, Theory of Change</a>.</li>`;
 
-  constructor(public api: ApiService, public institutionsSE: InstitutionsService, public rdPartnersSE: RdPartnersService, public rolesSE: RolesService) {}
+  constructor(
+    public api: ApiService,
+    public institutionsSE: InstitutionsService,
+    public rdPartnersSE: RdPartnersService,
+    public rolesSE: RolesService
+  ) {}
 
   institutions_institutions_type_name(partner) {
-    //('institutions_institutions_type_name');
     const insts = this.institutionsSE.institutionsList;
-    const institutionFinded = insts.find(institution => institution.institutions_id == partner.user_matched_institution.institutions_id);
-    partner.user_matched_institution.institutions_type_name = institutionFinded?.institutions_type_name;
+
+    const institutionFinded = insts.find(institution => institution.institutions_id == partner.institutions_id);
+
+    partner.obj_institutions.obj_institution_type_code.name = institutionFinded?.institutions_type_name;
+    partner.obj_institutions.website_link = institutionFinded?.website_link;
+  }
+
+  generateDescription(partner) {
+    const confidenceLevel = partner.result_kp_mqap_institution_obj.confidant;
+
+    if (confidenceLevel > 90) {
```

Comment:
isn't the confidence level supposed to be variable? @xKeCo

Reply:
Yes, that is correct. This development is associated with User Story #2, which is still in progress. Once this activity is completed, this value will become dynamic. @gcmartinezg